core/che-core-api-model/src/main/java/org/eclipse/che/api/core/model/workspace/devfile/Component.java

```diff
@@ -28,6 +28,12 @@
   /** Returns the plugin/editor FQN. Is mandatory only for cheEditor/chePlugin components types. */
   String getId();
 
+  /**
+   * Returns the preferences of the plugin. Example value of preference: `java.home:
```

Comment:
Backticks are not supported in javadoc afaik. Use `{@code}` for the intended effect.

Reply:
Used {@code}